apps/pollbook/backend/src/local_store.ts

```diff
@@ -364,56 +371,23 @@ export class LocalStore extends Store {
             SELECT v.voter_id, v.voter_data
             FROM voters v
             WHERE updated_last_name REGEXP ?
+              AND updated_middle_name REGEXP ?
               AND updated_first_name REGEXP ?
+              AND updated_suffix REGEXP ?
             `,
       `${lastNamePattern}`,
-      `${firstNamePattern}`
+      `${middleNamePattern}`,
+      `${firstNamePattern}`,
+      `${suffixPattern}`
     ) as Array<{ voter_id: string; voter_data: string }>;
 
     if (voterRows.length === 0) {
       return [];
     }
 
-    // Map voter rows to voter objects
```

Comment:
I realized we still need to apply events on top of `voter` objects even if we're not filtering by the values. It's readded in a later commit